### PR TITLE
WIP: lal: fixed configure arguments

### PR DESCRIPTION
**Pull request by duncanmmacleod**
#### Description

This PR attempts to fix the build failures introduced by #3197 (thanks to @cjones051073 for pointing this out to me). This is 'WIP' (at the time of submission) because I'm not sure whether the same change needs to be made in _all_ of the lalsuite subpackages, or just `lal`.

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14 18A391
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
Seems OK to me. Build failures now look like travis issues (mostly timeouts) not issues with the port.

File: science/lal/Portfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ PortSystem    1.0
 
 name          lal
 version       6.19.0
+revision      1
```
Comment (cjones051073) [inline science/lal/Portfile]: small detail, but the revision bump is not needed here as the port previously did not build, and you are making other changes to the port (that themselves would trigger a rebuild).

**Comment by duncanmmacleod** [comment]
@cjones051073, I'm preparing another suite of commits which replicate the changes made to `lal` across the other lalsuite subpackages. I'll remove the 'WIP' when I'm done. @skymoo should review all of this before merging.

**Comment by duncanmmacleod** [comment]
@skymoo, other than the failing CI build, I think this is ready for review.

**Comment by skymoo** [comment]
Changes look good to me, I'll run a local test build.

**Pull request merged**